parts/kubernetesmasteraddons-tiller-service.yaml

```diff
@@ -0,0 +1,29 @@
+# Produced by 'helm init --dry-run --debug'
```

Comment:
Can we please clean this manifest up given that most of it is null. It also needs the following label for addonmanager to pick it up

```
  labels:
    kubernetes.io/cluster-service: "true"
```